test/unit/tasks/download-assets.test.js

```diff
@@ -142,15 +142,16 @@ beforeEach(() => {
     }
   })
 })
-beforeAll(() => {
-  fs.mkdirsSync(tmpDirectory)
+beforeAll(async () => {
+  await fs.mkdir(tmpDirectory, { recursive: true })
 })
 
 afterAll(() => {
-  fs.removeSync(tmpDirectory)
+  // Couldn't get `fs.promises.rm` to work without permissions issues
```

Comment:
Very specific to windows - probably never going to be an issue on unix based systems.